script/hassfest/manifest.py

```diff
@@ -15,6 +15,14 @@
 DOCUMENTATION_URL_EXCEPTIONS = {"https://www.home-assistant.io/hassio"}
 
 SUPPORTED_QUALITY_SCALES = ["gold", "internal", "platinum", "silver"]
+SUPPORTED_IOT_CLASSES = [
+    "Assumed State",
+    "Calculated",
```

Comment:
I think that we shouldn't have Calculated for "internal" integrations. I think that they should just have none.

Reply:
I'm not sure what that means?
Calculated means: 

> The integration does not handle communication on it's own, but calculates the state.

Let's consider `sun`... its internal, but its state is clearly calculated. Or the `updater` integration, is internal, but is cloud polling.

I'm not sure why we would hide that?

Reply:
Yeah, the `sun` is fine. So is `updater`. But setting `default_config` to calculated seems wrong. I would even argue that `counter` should not be `calculated` because it's user input.